building-models/searching.md

```diff
@@ -21,4 +114,33 @@ echo 3 |\
 [3,2,1]
 ```
 
+Use the Nextmv cli to look at complete examples of more advanced search
+problems:
+
+```bash
+$ nextmv sdk init -t knapsack
+Successfully generated the knapsack template.
+$ nextmv sdk init -t sudoku
+Successfully generated the sudoku template.
+$ nextmv sdk init -t shift-scheduling
+Successfully generated the shift-scheduling template.
+```
+
+## Exercises
+
+* Run the turorial for different values of `n`, such as 4.
```

Comment:
```suggestion
* Run the tutorial for different values of `n`, such as 4.
```
typo

Reply:
I think `tuROARial` would make a great name for tutorials that we are very proud of 😆